File: Level 1/eCitizen.cpp
#include <iostream>
#include <string>

int main() {
    std::string validUser = "adminKE";
    std::string validPass = "254Secure";

    std::string userInput, passInput;

    std::cout << "Enter Username: ";
    std::cin >> userInput;

    std::cout << "Enter Password: ";
    std::cin >> passInput;

    if (userInput == validUser && passInput == validPass) {
        std::cout << "Access Granted" << std::endl;
    } else {
        std::cout << "Invalid Credentials" << std::endl;
    }

    return 0;
}

File: Level 1/eCitizen.py
VALID_USER = "adminKE"
VALID_PASS = "254Secure"

def login():
    user_input = input("Enter Username: ")
    pass_input = input("Enter Password: ")

    if user_input == VALID_USER and pass_input == VALID_PASS:
        print("Access Granted")
    else:
        print("Invalid Credentials")

if __name__ == "__main__":
    login()